package.json

```diff
@@ -73,6 +73,7 @@
     "pg": "8.11.3",
     "reflect-metadata": "^0.2.2",
     "serialize-error": "8.1.0",
+    "typescript-eslint": "^8.0.0-alpha.30",
```

Comment:
Check this--why wasn't it there before?

Reply:
Moved this dependency to the templates' package.json files